source/solution-utils/helpers.ts

```diff
@@ -9,3 +9,28 @@
 export function isNullOrWhiteSpace(str: string): boolean {
   return !str || str.replace(/\s/g, "") === "";
 }
+
+/**
+ * Determine if this appears to be json or a plain string
+ * @param str Input string to evaluate
+ * @returns Either a json object or a plain string
+ */
+export function parseJson(str) {
+  try {
+    return JSON.parse(str);
+  } catch (e) {
+    return [str];
+  }
+}
+
+/**
+ * Given a string that may describe actions in an image request, create a regex from the string
+ * @param matchPattern string to operate on
+ * @returns regex
+ */
+export function generateRegExp(matchPattern) {
+  const patternStrings = matchPattern.split("/");
+  const flags = patternStrings.pop();
+  const parsedPatternString = matchPattern.slice(1, matchPattern.length - 1 - flags.length);
+  return new RegExp(parsedPatternString, flags);
```

Comment:
I realize this feature is already out there, but do we really need regex to solve this problem?
It looks like these values are controlled in the lambda env, but are we worried about customers redos-ing themselves?

Reply:
Your comment is important but I think it is not relevant to this pull request. Only the location of the code has changed, the logic has not changed.